### ISSUE TITLE
Full width operator tool prototype

### ISSUE THREAD

**Issue by rymorgan** [reactioncommerce/reaction-feature-requests#50]
**Goal -** take our current operator experience and make it 100% view across every view. 

**Two reasons -** 
1. For storefronts that are using the graph ql starter kit it's confusing to have a second storefront in view. 
2. To maximize screen real estate as we move forward for larger ecommerce builds (more products, more features etc)

**Comment by rymorgan** [comment]
@spencern As discussed this issue might be best suited for @kieckhafer as a starter issue as he comes back + he has old-school knowledge of the Reaction to make implementation a little easier.

**Comment by brent-hoover** [comment]
I heartily endorse this project. I think it also makes it clearer that the PDP is no longer a WYSIWYG editor.

**Comment by rymorgan** [comment]
@zenweasel Yes! I see this issue as a first step and experiment to move towards a 100% width operator experience. Frankenstein, though it may be at first.
